bin/bcoin-cli

```diff
@@ -127,6 +127,14 @@ class CLI {
     this.log(filter);
   }
 
+  async estimateFee() {
+    const blocks = this.config.uint(0, 1);
+
+    const fee = await this.client.estimateFee(blocks);
+
+    this.log(fee);
```

Comment:
Should log the BTC amount instead of the direct JSON?

Reply:
Agreed, all the CLI methods operate on BTC and BTC/kB, like the RPC, but unlike the cURL and JS methods of the HTTP API 🤷‍♂ . Updated!